keyboards/primekb/prime_e/prime_e.c

```diff
@@ -15,8 +15,20 @@
  */
 #include "prime_e.h"
 
-void led_set_kb(uint8_t usb_led) {
-	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+void keyboard_pre_init_kb(void) {
+    setPinOutput(B2);
+    setPinOutput(B1);
+    setPinOutput(B3);
 
-	led_set_user(usb_led);
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(B2, led_state.num_lock);
+        writePin(B1, led_state.caps_lock);
+        writePin(B3, led_state.scroll_lock);
+    }
+    return res;
 }
```

Comment:
Cant do suggestions, but this should be removed, and the following added to `config.h`

```c
#define LED_NUM_LOCK_PIN B2
#define LED_CAPS_LOCK_PIN B1
#define LED_SCROLL_LOCK_PIN B3
```

Reply:
Solved by [`bb11569`](https://github.com/qmk/qmk_firmware/pull/11674/commits/bb11569095ab362aa156259efa5493243213cfde).